packages/test/src/test-integration.ts

```diff
@@ -197,12 +197,13 @@ if (RUN_INTEGRATION_TESTS) {
   });
 
   test('Worker default ServerOptions are generated correctly', async (t) => {
-    const client = new Connection();
-    const workflow = client.workflow<ArgsAndReturn>('args-and-return', { taskQueue: 'test' });
-    await workflow.start('hey', undefined, Buffer.from('abc'));
+    const client = new WorkflowClient();
+    const workflow = client.stub<ArgsAndReturn>('args-and-return', { taskQueue: 'test' });
+    const runId = await workflow.start('hey', undefined, Buffer.from('abc'));
+    await workflow.result();
     const execution = await client.service.getWorkflowExecutionHistory({
-      namespace: workflow.options.namespace,
-      execution: { workflowId: workflow.workflowId, runId: workflow.runId },
+      namespace: 'default',
```

Comment:
This default could be constant re-used in all these places

Reply:
SGTM